homeassistant/components/awair/const.py

```diff
@@ -0,0 +1,120 @@
+"""Constants for the Awair component."""
+
+from collections import namedtuple
+from datetime import timedelta
+import logging
+
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_BILLION,
+    CONCENTRATION_PARTS_PER_MILLION,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_ILLUMINANCE,
+    DEVICE_CLASS_TEMPERATURE,
+    TEMP_CELSIUS,
+    UNIT_PERCENTAGE,
+)
+
+AwairResult = namedtuple("AwairResult", ["device", "air_data"])
+
+API_CO2 = "carbon_dioxide"
+API_DUST = "dust"
+API_HUMID = "humidity"
+API_LUX = "illuminance"
+API_PM10 = "particulate_matter_10"
+API_PM25 = "particulate_matter_2_5"
+API_SCORE = "score"
+API_SPL_A = "sound_pressure_level"
+API_TEMP = "temperature"
+API_TIMEOUT = 20
+API_VOC = "volatile_organic_compounds"
+
+ATTRIBUTION = "Awair air quality sensor"
+
+ATTR_ICON = "icon"
+ATTR_LABEL = "label"
+ATTR_UNIT = "unit"
+
+# Some of these names are historical, because this integration
+# created sensors with names that matched the API somewhat
+# directly. If air quality device classes are standardized
+# across home assistant, then we should adjust these as well.
+# For example:
+# score -> air_quality_index
+# CO2   -> carbon_dioxide
+# PM10  -> particulate_matter_10
+# PM2.5 -> particulate_matter_2_5
+# VOC   -> volatile_organic_compounds
+
+DEVICE_CLASS_AIR_QUALITY_INDEX = "score"
+DEVICE_CLASS_CO2 = "CO2"
+DEVICE_CLASS_PM_10 = "PM10"
+DEVICE_CLASS_PM_2_5 = "PM2.5"
+DEVICE_CLASS_SOUND_LEVEL = "sound_level"
+DEVICE_CLASS_VOLATILE_ORGANIC_COMPOUNDS = "VOC"
+
+DOMAIN = "awair"
+
+DUST_ALIASES = [API_PM25, API_PM10]
+
+LOGGER = logging.getLogger(__package__)
+
+UPDATE_INTERVAL = timedelta(minutes=5)
+
+SENSOR_TYPES = {
+    API_SCORE: {
+        ATTR_DEVICE_CLASS: DEVICE_CLASS_AIR_QUALITY_INDEX,
+        ATTR_ICON: "mdi:blur",
+        ATTR_UNIT: UNIT_PERCENTAGE,
+        ATTR_LABEL: "Awair Score",
+    },
+    API_HUMID: {
+        ATTR_DEVICE_CLASS: DEVICE_CLASS_HUMIDITY,
+        ATTR_ICON: "mdi:water-percent",
```

Comment:
Entities with device class set should not set a custom icon.
```suggestion
        ATTR_ICON: None,
```

Reply:
Will setting it to `None` cause the home assistant to display the default icon? That wasn't clear to me when writing the code, hence why I set one.

Reply:
Yes, it will use the default icon for that device class.